docs-app/tests/acceptance/button-test.gts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-undef -- Until https://github.com/ember-cli/eslint-plugin-ember/issues/1747 is resolved... */
+/* eslint-disable simple-import-sort/imports,padding-line-between-statements,decorator-position/decorator-position -- Can't fix these manually, without --fix working in .gts */
+import { module, test } from 'qunit';
+import { click, visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+
+
+module('Acceptance | Button', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting the button page', async function (assert) {
+
+    await visit('/docs/components/button')
+    assert.strictEqual(currentURL(), '/docs/components/button');
+
+    assert.dom('h1').hasText('Button');
+    assert.dom('.docfy-demo__example').hasTagName('div');
+
+    assert.dom('.docfy-demo__example button').hasText('Button');
```

Comment:
It's possible to click the button here, and an `alert` will pop up, but I didn't know how to select it or hit the "okay" button so I omitted trying this.